crypto/crypto.go

```diff
@@ -62,32 +58,29 @@ func StringToSignatureAlgorithm(s string) SignatureAlgorithm {
 }
 
 // HashAlgorithm is an identifier for a hash algorithm.
-type HashAlgorithm int
+type HashAlgorithm hash.HashingAlgorithm
 
 const (
-	UnknownHashAlgorithm HashAlgorithm = iota
-	SHA2_256
-	SHA2_384
-	SHA3_256
-	SHA3_384
+	UnknownHashAlgorithm HashAlgorithm = HashAlgorithm(hash.UnknownHashingAlgorithm)
+	SHA2_256                           = HashAlgorithm(hash.SHA2_256)
+	SHA2_384                           = HashAlgorithm(hash.SHA2_384)
+	SHA3_256                           = HashAlgorithm(hash.SHA3_256)
+	SHA3_384                           = HashAlgorithm(hash.SHA3_384)
```

Comment:
I would like to remove the `384` versions as they are not used in Flow, but that's also considered a breaking change.